.changeset/itchy-bobcats-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
```

Comment:
Oh another minor change while you’re at it: I think this should be a `patch` not a minor

Reply:
I see... Was a bit confused about that. Iirc `yarn changeset` only gave me an option between major and minor.
Would just editing the file fix it? Or do I need to run a specific command and delete the current file? (I'm not familiar with changesets as a whole, a great opportunity to learn)

Reply:
Added in 7393eac